LoopCore/NSUserDefaults.swift

```diff
@@ -21,7 +21,9 @@ extension UserDefaults {
         case insulinSensitivitySchedule = "com.loudnate.Naterade.InsulinSensitivitySchedule"
     }
 
-    var basalRateSchedule: BasalRateSchedule? {
+    public static let appGroup = UserDefaults(suiteName: Bundle.main.appGroupSuiteName)
```

Comment:
This property is now an Optional when it wasn't before—is there a known occasion when we expect to have `nil` here?